lib/spack/docs/known_issues.rst

```diff
@@ -35,3 +35,36 @@ A workaround is to explicitly activate the variant related to the dependency:
    $ spack install hdf5+mpi ^openmpi
 
 See https://github.com/LLNL/spack/issues/397 for further details.
+
+
+---------------------------------
+``spack extensions`` doesn't work
+---------------------------------
+
+**Status:** Up for grabs if you want to try to fix it
+
+Spack provides an ``extensions`` command that lists all available extensions
+of a package, the ones that are installed, and the ones that are already
+activated. This is very useful in conjunction with ``spack activate``.
+Unfortunately, this command no longer works:
+
+.. code-block:: console
+
+   $ spack extensions python
+   ==> python@2.7.13%clang@8.0.0-apple~tk~ucs4 arch=darwin-sierra-x86_64 -ckrr4mg has no extensions.
+
+
+See https://github.com/LLNL/spack/issues/2895 for further details.
+
+
+----------------------------
+``spack setup`` doesn't work
+----------------------------
```

Comment:
@tgamblin  Instead of adding this text, let's just merge #2664.  This bug fix is now almost 4 months old.  I addressed the issues brought up in the review, and yet still no action on the merge.  Very frustrating...